ed448/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "ed448"
```

Comment:
We don't own the `ed448` crate

Reply:
Exactly.
What should I name?
Is `ed448-sig` fine?

Reply:
Hi @tarcieri 
Is everything okay?

Reply:
I can't think of a good name here, sorry. Especially since it would be different from `ed25519`

Reply:
I see.
`ed448_goldilocks` is still available.

Reply:
Doesn’t look like it: https://crates.io/crates/ed448-goldilocks

Reply:
I would say that is `ed448-goldilocks` but `ed448_goldilocks` is available.

Reply:
Ah
It seems it's not available.

Reply:
hyphen and underbar are not distinguished.

Reply:
getting eddsa and integrate ed448 and ed25519

Reply:
in root directly, there are dsa and ecdsa.

Reply:
I think it makes sense

Reply:
eddsa seems available miraculously
https://crates.io/crates/eddsa

Reply:
Good call! Snagged it

Reply:
Cool!
How can I proceed this PR?
There are two task.

- implement ed448
- integrate ed448 and ed25519 to eddsa

I think it's better to create PR separately because ed448 is enough big changes.

Reply:
We still need to come up with an Ed448 crate

Reply:
Copy that.
I will post here once I come up with.

Reply:
How about edwards448?
https://crates.io/crates/edwards448

Reply:
You have edwards25519
https://crates.io/crates/edwards25519

Reply:
In rfc, it's described as edwards448 and edwards25519
https://datatracker.ietf.org/doc/html/rfc8032

Reply:
I went ahead and snagged `edwards448`.

The problem is those describe the Edwards form of the elliptic curve, not the digital signature algorithm.

Reply:
I think I can generalize ed448 and ed25519 signature because this library doesn't have curve arithmetic.
How about implementing `signature/eddsa` which supports edwards curve dsa as the same with ecdsa and importing eddsa from `elliptic-curve/edwards25519` and `elliptic-curve/edwards448`?

Reply:
We don't currently have or plan to maintain implementations of Curve25519 or Ed448-Goldilocks.

`ed25519` is effectively finished and it would be pretty painful to make any additional breaking changes at this point. We're finally to the place where almost everything has been updated to v2.0 which has taken most of 2023.

Reply:
Copy that.
Let me think.

Reply:
How about eddsa448 and renaming ed25519 to eddsa25519?
https://crates.io/search?q=eddsa448

Reply:
Again, it’s taken almost a year to wrap up the latest `ed25519` upgrade and the work is still incomplete.

I’d really rather avoid making any additional breaking changes to `ed25519` for at least another year after everything is fully upgraded and usable, especially just for supporting Ed448 which is relatively obscure.

Also “EdDSA25519” is not the construction’s name, it’s Ed25519.

Reply:
I guess just call it `ed448-signature` for now. I can't think of a better name.

Reply:
Thank you for the discussion.
I named it ed448-signature.